### DISCUSSION TITLE
Can we add a new backup endpoint to store only single backup file?

### DISCUSSION THREAD

**Discussion by johnnydev543** [OffchainLabs/prysm#10546] [General]
The db backup distinguished by date doesn't make any sense, as we always want the latest db backup on local machine if we don't want to waste of time sync it from the internet. We won't restore the db backup from the one other than the latest we have.
Or perhaps the db is forked, but if it's forked, we wouldn't wasted of time finding out the db backup files which is not forked, just resync it.
So, maybe add a new endpoint on `/db/backup/single`?
If we do this, this will help when we just want a single backup file every time the endpoint been triggered by a cron job.
What do you think?

**Comment by prestonvanloon** [comment]
Now that checkpoint sync is available, you may consider using a cron job to save checkpoints over time and then serve those from a custom solution or existing beacon node.

https://docs.prylabs.network/docs/prysm-usage/checkpoint-sync
